lib/pact_broker/pacts/service.rb

```diff
@@ -137,10 +136,9 @@ def create_pact params, version, provider
 
       def trigger_webhooks pact
         # TODO add tests for this
+        webhook_service.trigger_webhooks pact, nil, PactBroker::Webhooks::WebhookEvent::CONTRACT_PUBLISHED
         if pact_is_new_or_pact_has_changed_since_previous_version?(pact)
           webhook_service.trigger_webhooks pact, nil, PactBroker::Webhooks::WebhookEvent::CONTRACT_CONTENT_CHANGED
-        else
-          logger.debug "Pact has not changed since previous version, not triggering webhooks"
         end
```

Comment:
This line is still required. You can change the text to "not triggering webhooks for changed content".